variablesview.py

```diff
@@ -299,9 +298,46 @@ def on_item_double_click(self, index):
         else:
             self.object_picked.emit(name)
 
-    def on_item_right_click(self, index):
-        name = index.data(Role_Name)
-        parent = index.data(Role_Parent)
+    def _open_menu(self, position):
+
+        menu = QMenu()
+        var_val_action = QAction("Copy Variable Value", self)
```

Comment:
```suggestion
        var_val_action = QAction("Copy Variable Value", menu)
```

(Elsewhere too)
Since we build the menu from scratch with each right click, the actions we create should be parented to the menu